library/src/main/java/com/getstream/sdk/chat/view/MessageListView.java

```diff
@@ -243,6 +243,61 @@ public Drawable getDrawableForAttachment(Message message, Boolean mine, List<Mes
                         .cornerRadii(topLeftRadius, topRightRadius, bottomRightRadius, bottomLeftRadius)
                         .build();
             }
+
+            @Override
+            public Drawable getDrawableForAttachmentDescription(Message message, Boolean mine, List<MessageViewHolderFactory.Position> positions){
```

Comment:
The method seems like almost a copy paste from other methods of `BubbleHelper`. Let's extract `BubbleHelper` implementation into package private class, might be something like  `DefaultBubbleHelper.java`.  It will eliminate 200 lines from `MessageListView`.

Views easily become bloated with similar calls and repetitive logic.